go.mod

```diff
@@ -0,0 +1,4 @@
+module github.com/wangkuiyi/gotorch
+
+go 1.13
```

Comment:
Let us don't introduce Go module. I think we dont' need it.

Reply:
Done, remove it.